dashboard/lib/pd/payment/payment_calculator_base.rb

```diff
@@ -62,13 +62,13 @@ def get_pay_period(workshop)
 
     # Determines the payment type.
     # Override in derived classes. Default is either "PLP Urban" or "PLP Non-urban"
```

Comment:
No. These are still supposed to say "PLP" at least until we upgrade the payment model later this sprint (in which case the PLP reference will go away among other things)